domains/client/domain-operator/src/domain_bundle_proposer.rs

```diff
@@ -142,12 +149,28 @@ where
                 let next_estimated_bundle_weight =
                     estimated_bundle_weight.saturating_add(tx_weight);
                 if next_estimated_bundle_weight.any_gt(domain_block_limit.max_block_weight) {
-                    break;
+                    if skipped < MAX_SKIPPED_TRANSACTIONS
+                        && Percent::from_rational(
+                            estimated_bundle_weight.ref_time(),
+                            domain_block_limit.max_block_weight.ref_time(),
```

Comment:
I have a question here should this not be `max_bundle_weight` instead of `max_block_weight`? Since, operator can produce multiple bundles per block.

Reply:
You're right, but we don't have a bundle weight limit yet (it's same as block weight limit rn), it's TODO tracked by #2413